Can't make

I've installed `node` on my server (I usually use Python Flask for backend and vanilla JS for front-end), and managed to run `npm install` without errors. 

Running `make` throws the following error:
```
make
mkdir -p ./build
touch ./build/.exists--used_by_Makefile
rm -rf build/font
cp -r src/font build/font
./node_modules/.bin/lessc  ./src/css/main.less > build/mathquill.css
make: *** [build/mathquill.css] Error 1
```
Any way to fix this?

> I'd honestly prefer to have a distribution of mathquill that I could download, unzip, and be ready to go.

There is, at the bottom of https://github.com/mathquill/mathquill/releases :

![pasted image at 2016_03_10 02_25 pm](https://cloud.githubusercontent.com/assets/225809/13730535/3ad804fe-e90f-11e5-9b47-b06cf55c2ab6.png)

> ```
> make: *** [build/mathquill.css] Error 1
> ```
That's due to one of the shell commands that `make` is running exiting with a nonzero exit code, can you try running the individual commands and seeing what the actual error message is?

Well, after I run that command, it throws an error on this one:
```
cat src/intro.js node_modules/pjs/src/p.js src/tree.js src/cursor.js src/controller.js src/publicapi.js src/services/parser.util.js src/services/saneKeyboardEvent
s.util.js src/services/exportText.js src/services/focusBlur.js src/services/keystroke.js src/services/latex.js src/services/mouse.js src/services/scrollHoriz.js s
rc/services/textarea.js src/commands/math.js src/commands/text.js src/commands/math/advancedSymbols.js src/commands/math/basicSymbols.js src/commands/math/command
s.js src/commands/math/LatexCommandInput.js src/outro.js build/.exists--used_by_Makefile | ./script/escape-non-ascii > build/mathquill.js
make: *** [build/mathquill.js] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,23 @@
 #
+# -*- Prerequisites -*-
+#
+
+# the fact that 'I am Node.js' is unquoted here looks wrong to me but it
+# CAN'T be quoted, I tried. Apparently in GNU Makefiles, in the paren+comma
+# syntax for conditionals, quotes are literal; and because the $(shell...)
+# call has parentheses and single and double quotes, the quoted syntaxes
+# don't work (I tried), we HAVE to use the paren+comma syntax
+ifneq ($(shell node -e 'console.log("I am Node.js")'), I am Node.js)
+  ifeq ($(shell nodejs -e 'console.log("I am Node.js")' 2>/dev/null), I am Node.js)
+    $(error You have /usr/bin/nodejs but no /usr/bin/node, please 'sudo apt-get install nodejs-legacy' (see http://stackoverflow.com/a/21171188/362030 ))
+  endif
+
+  $(error Please install Node.js: https://nodejs.org/ )
+endif
+
+
+
+#
 # -*- Configuration -*-
 #
 

